bin/sl-run.js

```diff
@@ -17,6 +17,14 @@ var agentOptions = {
   logger: config.logger,
 };
 
+var tracer = require('concurix');
+var tracerOptions = {
+  accountKey: agent().config.appName,
```

Comment:
agent() shouldn't be called if we aren't profiling, its a "lazy loader" (see `lib/agent.js`). I'd suggest wrapping this up as:
```
function trace() {
   var tracerOptions = {
     // ...
  };
  tracer(traceOptions);
}
```

And calling `trace()` in the switch statement.